lib/logstash/filters/elasticsearch.rb

```diff
@@ -84,7 +84,7 @@ def filter(event)
     begin
       query_str = event.sprintf(@query)
 
-      params = { :q => query_str, :size => result_size }
+      params = { :q => query_str, :size => result_size, :index => @index }
```

Comment:
does this mean that setting the index [here](https://github.com/purbon/logstash-filter-elasticsearch/blob/f9c3103c2ad91e5151c9cc0b0f01e0274fce667f/lib/logstash/filters/elasticsearch.rb#L78) does nothing?
if so we should remove it.

Also, is the default  `:index => ""` going to query all indices? Is it the same as `_all`?